app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -113,8 +113,8 @@ Teacher training fees for non-UK candidates vary between training providers.
 
 Fees cost:
 
-* an average of £14,765
-* a maximum of about £36,000
+* an average of $fees_pgitt_internationalaverage$
+* a maximum of about $fees_pgitt_internationalmaxfulltime$
```

Comment:
As above, the top amount is different of review app and live site

Reply:
Request from ITRU team to update